src/odemis/odemisd/mdupdater.py

```diff
@@ -262,12 +262,11 @@ def updateMDFromVA(val, md_key=md_key, comp_affected=comp_affected):
         return True
 
     def observeLight(self, light, comp_affected):
-
-        def updateInputWL(emissions, light=light, comp_affected=comp_affected):
+        def updateLightPower(power, light=light, comp_affected=comp_affected):
```

Comment:
I removed `updateInputWL` which include emissions update, and moved updating `model.MD_IN_WL` to `updateLightPower` 
@pieleric